components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml

```diff
@@ -8027,6 +8027,9 @@ components:
         isDefaultVersion:
           type: boolean
           example: false
+        organizationId:
+          type: string
+          example: lamda
```

Comment:
shall we replace this example?

Reply:
Fixed https://github.com/wso2/carbon-apimgt/pull/10242/commits/1460f7ae673d4205b2772bae36fef14e94bab6b1